test/index.test.js

```diff
@@ -1,6 +1,6 @@
-const { Application } = require('probot')
 // Requiring our app implementation
-const myProbotApp = require('..')
+import myProbotApp from '../src/index.js'
+const probot = require('probot')
```

Comment:
Why require instead of import?

Reply:
i put less than 100% effort into sanitizing the imports in the tests because babel seemed to be a little more forgiving to mixing the two styles. node was yelling loudly about it so i had to pull the changes through the whole src directory